cineast-runtime/src/main/java/org/vitrivr/cineast/standalone/run/GenericExtractionItemHandler.java

```diff
@@ -139,10 +139,12 @@ public Set<MediaType> getMediaTypes() {
     // #353: Respect the given segmenter
     final Set<MediaType> segmenterTypes;
     final Segmenter<Object> segmenter = context.newSegmenter();
-    segmenterTypes = segmenter.getMediaTypes();
-    segmenterTypes.forEach(t -> {
-      handlers.put(t, new ImmutablePair<>(handlers.get(t).getLeft(), () -> segmenter));
-    });
+    if (segmenter != null) {
```

Comment:
Can we add a log-message if the segmenter is null? I don't see the scenario where the segmenter is null but we have a proper config

Reply:
The segmenter is optional in the config, so there are plenty of scenarios where this would be the case. But sure, we can log this.